understanding/status-messages.html

```diff
@@ -55,317 +53,216 @@
-                        content to other pieces of information on the screen may provide a visual context
-                        that is lacking in the text alone.
+                     <p>表示テキストの追加だけでは、支援技術の利用者に十分な情報を伝えない可能性がある。例えば、新しいコンテンツが画面上の他の情報に近接していることで、テキストだけでは欠けている視覚的なコンテキストを提供してもよい。
                      </p>
                      		
-                     <p>In such cases, authors may wish to designate additional content for inclusion in the
-                        status message, including non-displayed text which can be provided to the assistive
-                        technologies, for added context. Important considerations regarding the appropriate
-                        use of such techniques are further discussed in the Sufficient Techniques.
+                     <p>このような場合、コンテンツ制作者は、追加されたコンテキストのために、支援技術に提供できる非表示テキストを含むステータスメッセージに含める追加コンテンツを指定することを望んでもよい。このような手法の適切な使用に関する重要な考慮事項は、十分なテクニックでさらに説明する。
                      </p>
                      				
                   </section>
                   				
                   <section id="modification-of-status-text">
                      		
-                     <h4>Modification of status text</h4>
+                     <h4>ステータステキストの変更</h4>
                      		
-                     <p>If a status message persists on the page, modifications to this text are usually equivalent
-                        to a new status message. An example would be a shopping cart which updates text from
-                        reading "0 items" to "3 items". Typical methods of writing such changes in the page
-                        content result in the entire modified text string being considered a new change, and
-                        thus read by assistive technologies. However, where only the number in this string
-                        was coded as an updated chunk of content, the resulting experience for screen reader
-                        users could be to only hear "three", which may not be sufficient information to provide
-                        context for the user. In such situations, marking the entire "3 items" string as the
-                        status text would normally be a better solution. See Sufficient Techniques for more
-                        discussion, including the use of <code>aria-atomic</code>. In this case it would also be a courtesy to add offscreen text such as "in shopping
-                        cart" to the message.
+                     <p>ステータスメッセージがページに存続している場合、このテキストの変更は、通常新しいステータスメッセージと同等である。例は「0アイテム」の読み取りから「3アイテム」に更新するショッピングカードだろう。ページコンテンツにこのような変更を書き込む一般的な方法は、修正されたテキスト文字列全体が新しい変更と見なされ、そして支援技術によって読まれる。しかしながら、この文字列内の数字だけがコンテンツの更新された塊としてコード化された場合、スクリーンリーダーの利用者にもたらされる体験は「3」と聞こえるだけであり、利用者にコンテキストを提供するのに十分な情報ではないだろう。このような状況では、「3アイテム」の文字列全体をステータステキストとして示すと、通常より良い解決策になるだろう。<code>aria-atomic</code>の使用を含むさらなる説明は、十分なテクニックを参照。この場合、「ショッピングカート内」などの画面外のテキストをメッセージに追加すると親切だろう。
```

Comment:
ショッピングカード → ショッピングカート

reading "0 items" は「0 items と読み取れるもの」というような意味かと。item も適切な日本語にした方がいいかもという気もしなくもないですが、まあいいか……。

offscreen text は画面外のテキストですが要は class="sr-only" みたいなやつですね。まあそのままでもいいか……。このセクション全体として、aria-atomicを知らない人が読んでも何の話かさっぱりわからないのではという感じがしますが、そこはまあ原文がそうだから仕方ないなぁと

```suggestion
                     <p>ステータスメッセージがページに存続している場合、このテキストの変更は、通常、新しいステータスメッセージと同等である。例としては、テキストの「0アイテム」という表示を「3アイテム」に更新するショッピングカートがある。一般的な方法でページコンテンツにこのような変更を書き込んだ場合、修正されたテキスト文字列全体が新しい変更と見なされ、そして支援技術によって読まれる。しかしながら、この文字列内の数字だけをコンテンツの更新されたチャンクとしてコード化した場合、スクリーンリーダーの利用者には「3」と聞こえるだけの体験がもたらされ、利用者にコンテキストを提供するのに十分な情報とはいえないだろう。このような状況では、「3アイテム」の文字列全体をステータステキストとして示すことが、通常は、より良い解決策になるだろう。<code>aria-atomic</code>の使用を含むさらなる説明は、十分なテクニックを参照。この場合、「ショッピングカート内」などの画面外のテキストをメッセージに追加すると親切だろう。
```